dts/bindings/sensor/invensense,icm20948.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2021 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  ICM20948 Motion sensor by invensense
+
+compatible: "invensense,icm20948"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      The INT signal default configuration is active-high. The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  accel-hz:
+    type: int
+    description: |
+      Default frequency of accelerometer. (Unit - Hz)
+      Maps to ACCEL_ODR field in ACCEL_CONFIG0 setting
+      Power-on reset value is 1000.
+    default: 1000
+    enum:
```

Comment:
The ODRs here don't seem to match the datasheet ODRs.

The ODRs for gyro differ from accel and mag. They also seem to depend on a filter choice flag and low power mode flag. At minimum I'd expect all the possible rates to be listed here but yeah.. this is one of those parts that has ODRs, power mode, and filter flags that are all dependent on each other.

Reply:
I was about to say just that :)
@lazaroofarrill it looks like you have some "TODO" where you'd like these binding properties actually used in your driver. It would be great to either not have these, I assume copy-pasted, properties for now, or add proper support in the driver already.

Reply:
Is there a way to make device tree property dependent on another?

Reply:
@kartben I've fixed the config file but it is still showing the "Changes requested label"